doc/mgr/smb.rst

```diff
@@ -952,3 +952,61 @@ at the prompt. Refer to the `smbclient documentation`_ for more details.
 
 .. _smbclient documentation:
    https://www.samba.org/samba/docs/current/man-html/smbclient.1.html
+
+
+Updating QoS Settings for CephFS-Backed Shares
+=================================================
+
+The SMB manager module now supports updating *QoS (Quality of Service)* settings
+for CephFS-backed shares. This allows administrators to apply per-share rate
+limits on SMB I/O operations, specifically limits on the number of operations
+per second (IOPS) and bandwidth in bytes per second, for both read and
```

Comment:
`I/O (or Input/Output)` Operations per Second - see https://en.wikipedia.org/wiki/IOPS  (otherwise it would just be "OPS" :-) ).